libraries/rush-lib/src/logic/operations/OperationExecutionManager.ts

```diff
@@ -216,11 +220,14 @@ export class OperationExecutionManager {
   /**
    * Handles the result of the operation and propagates any relevant effects.
    */
-  private _onOperationComplete(record: OperationExecutionRecord): void {
+  private _onOperationComplete(record: OperationExecutionRecord, executionQueue: AsyncOperationQueue): void {
```

Comment:
May as well store the AsyncOperationQueue on `this`, since we are already tracking state via the `_completedOperations` and related properties.

Reply:
`_completedOperations` in OperationExecutionManager has subtle difference with `_completedOperation` in AsyncOperationQueue. That's `_completedOperations` in OperationExecutionManger excludes those "no-op" Operations (runner.slient is true)